caiman/base/movies.py

```diff
@@ -17,6 +17,7 @@
 from IPython.display import display, Image
 import ipywidgets as widgets
 import logging
+import matplotlib.pyplot as plt
```

Comment:
I think this new import is not needed? doesn't seem to be used anyhow

Reply:
Good eye I was plotting something in an old debugging step.